src/Magnum/Audio/Buffer.h

```diff
@@ -83,7 +89,27 @@ class Buffer {
              *
              * @requires_al_extension Extension @al_extension{EXT,double}
              */
-            StereoDouble = AL_FORMAT_STEREO_DOUBLE_EXT
+            StereoDouble = AL_FORMAT_STEREO_DOUBLE_EXT,
+
+            Quad8 = AL_FORMAT_QUAD8,
```

Comment:
@Squareys I did add them to Extensions.h; did you miss them?

I can add some notes though, sure.